loopout.go

```diff
@@ -693,6 +715,10 @@ func (s *loopOutSwap) waitForHtlcSpendConfirmed(globalCtx context.Context,
 				return nil, err
 			}
 
+			// If we successfully published a spend, we push the
+			// preimage to the server.
```

Comment:
So we do it every block. Sounds okay, but is there another reason that this is a good interval besides that it is easy to implement?

Reply:
If we are only going to reveal our preimage after we have revealed it through a sweep, after a sweep attempt is the only time we need to re-evaluate whether to push again? I tried it with its own ticker, but then we just have to wait for the `timerChan` to tick first, followed by the preimge push ticker (which just makes htlc claim slower) 

There is the off chance that the sever fails, but I don't think we should form our logic based on the server maybe being down? And if it is, every block is a reasonable enough random retry delay.

Reply:
Yes it is also worth something to have only a single timer

Reply:
Seems that the current version of the pr does have the separate timer?

Reply:
Yeah I changed my mind after testing and seeing the double wait, should have marked as do not review yet, sorry!